tests/unit/BenCorrection.spec.ts

```diff
@@ -6,7 +6,7 @@ import sinon from 'sinon'
 import { getVuexStore } from '@/store/'
```

Comment:
This is the original Correction.spec.ts file, except it imports/mounts BenCorrection.vue instead of Correction.vue.